reactos/drivers/usb/usbhub/fdo.c

```diff
@@ -2009,11 +2063,13 @@ USBHUB_FdoHandlePnp(
         case IRP_MN_QUERY_REMOVE_DEVICE:
         case IRP_MN_QUERY_STOP_DEVICE:
         {
+            DPRINT("IRP_MN_QUERY_STOP_DEVICE\n");
             Irp->IoStatus.Status = STATUS_SUCCESS;
             return ForwardIrpAndForget(DeviceObject, Irp);
         }
         case IRP_MN_REMOVE_DEVICE:
         {
+            DPRINT("IRP_MN_REMOVE_DEVICE\n");
             Irp->IoStatus.Status = STATUS_SUCCESS;
             IoCompleteRequest(Irp, IO_NO_INCREMENT);
```

Comment:
This is supposed to be ForwardIrpAndForget too (and make sure to return its Status value). Not sure why DV isn't complaining